### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by antonchanning** [blockpress/blockpress.me#54]
Add logo to blockpress in several places...

# Loading graphic...
Replace the word 'loading' in index.html with an animation of the blockpress logo. Could be gif, but should have a fairly small file size. Alt text should say 'loading blockpress'.

# Favicons
Add various sized png logos as favicons to the site. Use prefered method as defined by w3c: https://www.w3.org/2005/10/howto-favicon

To see how size variants are supported, see this code from steemit.com:

	<link rel="icon" type="image/x-icon" href="/favicon.ico?v=2" data-reactid="22"/>
	<link rel="apple-touch-icon-precomposed" sizes="152x152" href="/images/favicons/apple-touch-icon-152x152.png" type="image/png" data-reactid="30"/>
	<link rel="icon" type="image/png" href="/images/favicons/favicon-196x196.png" sizes="196x196" data-reactid="31"/>
	<link rel="icon" type="image/png" href="/images/favicons/favicon-96x96.png" sizes="96x96" data-reactid="32"/>
	<link rel="icon" type="image/png" href="/images/favicons/favicon-32x32.png" sizes="32x32" data-reactid="33"/>
	<link rel="icon" type="image/png" href="/images/favicons/favicon-16x16.png" sizes="16x16" data-reactid="34"/>
	<link rel="icon" type="image/png" href="/images/favicons/favicon-128.png" sizes="128x128" data-reactid="35"/>

We might not want as many as that.

**Comment by antonchanning** [comment]
Also, we'll probably want ours to be in a different folder path. I'd suggest `/content/default/logo/`.
